api/pubsub.py

```diff
@@ -70,10 +70,19 @@ async def publish(self, channel, message):
         await self._redis.publish(channel, message)
 
     async def publish_cloudevent(self, channel, data, attributes=None):
+        """
+        We receive dict with message key in attributes argument.
+            i.e. attributes = {'message': ''}
+        As per CloudEvents documentation for using CloudEvent contructor:
+            attributes : dict with type and source keys
+            data : dict with message key
+        """
+        event_data = attributes
+        attributes = None
         if attributes is None:
             attributes = {
                 "type": "api.kernelci.org",
                 "source": self._settings.cloud_events_source,
             }
-        event = CloudEvent(attributes=attributes, data=data)
+        event = CloudEvent(attributes=attributes, data=event_data)
```

Comment:
Here is what we are receiving in publish_cloudevent function arguments:
```
attributes {'message': 'hii'}
data {'specversion': '1.0', 'id': '8cf20531-f441-4ce3-b08d-02229f2b9b6f', 'source': 'https://api.kernelci.org/', 'type': 'api.kernelci.org', 'time': '2021-12-09T09:32:33.784288+00:00'}
```
Now, CloudEvent constructor will have two parameters : arguments & data.
In the arguments : A dictionary with type and source keys should be passed
In the data : A dictionary with message should be passed.

So I created event_data variable from the `attributes {'message': 'hii'}` functional argument. That leaves data argument unchanged in case we need to use it in the future.

I think we can drop the `if attributes is None:` condition and create another attributes dict with type and source from what we have received in `data {'specversion': '1.0', 'id': '8cf20531-f441-4ce3-b08d-02229f2b9b6f', 'source': 'https://api.kernelci.org/', 'type': 'api.kernelci.org', 'time': '2021-12-09T09:32:33.784288+00:00'}`